smqtk_iqr/web/search_app/modules/file_upload/FileUploadMod.py

```diff
@@ -16,7 +18,8 @@ class FileUploadMod (flask.Blueprint):
     Flask blueprint for file uploading.
     """
 
-    def __init__(self, name, parent_app, working_directory, url_prefix=None):
+    def __init__(self, name: str, parent_app: IqrSearchDispatcher,
```

Comment:
I'm also unsure about the types of `parent_app` throughout the mypy updates

Reply:
I think in most contexts it is `IqrSearchDispatcher`. In this case that is the specific type that presents the `module_login` property that is used later with route setup.